extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthProcessor.java

```diff
@@ -67,16 +87,35 @@ void build(SmallRyeHealthTemplate template, RecorderContext recorder,
             BuildProducer<AdditionalBeanBuildItem> additionalBean,
             BuildProducer<BeanDefiningAnnotationBuildItem> beanDefiningAnnotation) throws IOException {
 
+        //String livenessPath = health.healthPath + health.livenessRelPath;
+        //String readinessPath = health.healthPath + health.readinessRelPath;
```

Comment:
Either use them or remove them.

If we can use recursive resolution as I mentioned above, it would simplify all of this.

Reply:
forgot them, sorry